lib/orders/domain/beverage.rb

```diff
@@ -0,0 +1,13 @@
+require 'orders/domain/invalid_beverage_error'
+
+module Orders
+  module Domain
+    class Beverage
```

Comment:
You don't need this kind of inheritance tree in Ruby. So long as the method exists, and responds in the correct way (no refused bequests, then you are good!)